fhir-server-test/src/test/java/com/ibm/fhir/server/test/ExportOperationTest.java

```diff
@@ -26,68 +31,54 @@
 import com.ibm.fhir.model.format.Format;
 import com.ibm.fhir.model.generator.FHIRGenerator;
 import com.ibm.fhir.model.generator.exception.FHIRGeneratorException;
+import com.ibm.fhir.model.resource.Condition;
+import com.ibm.fhir.model.resource.Group;
+import com.ibm.fhir.model.resource.Group.Member;
+import com.ibm.fhir.model.resource.Observation;
 import com.ibm.fhir.model.resource.Parameters;
 import com.ibm.fhir.model.resource.Parameters.Parameter;
+import com.ibm.fhir.model.resource.Patient;
+import com.ibm.fhir.model.test.TestUtil;
 import com.ibm.fhir.model.type.Instant;
+import com.ibm.fhir.model.type.Reference;
 
 /**
- * These tests exercise the $export operation, a BulkData specification defined operation.
- * 
- * @author pbastide
+ * These tests exercise the $export operation, a BulkData specification defined operation
  *
  */
 public class ExportOperationTest extends FHIRServerTestBase {
-
     public static final String TEST_GROUP_NAME = "export-operation";
-
     public static final String PATIENT_VALID_URL = "Patient/$export";
-    public static final String GROUP_VALID_URL = "Group/$export";
+    public static final String GROUP_VALID_URL = "Group/?/$export";
     public static final String BASE_VALID_URL = "/$export";
-
     public static final String BASE_VALID_STATUS_URL = "/$export-status";
-
     public static final String FORMAT = "application/fhir+ndjson";
+    public static final boolean ON = true;
```

Comment:
we run these on each Integration test?

Reply:
let me set it to false. :)

Reply:
I didn't add ExportOperationTest to server integration test testng.xml, so it doesn't run in the pipeline now.  I prefer to run this in local integration test at present.  but let me add it to the pipeline, because seems the deployment/configurations are ready in the pipeline, so, let me try to enable it now and let's see what we can get...

Reply:
en, it failed in the pipeline, because there is unfinished db2 javabatch config in server.xml, let me remove it from server.xml.

Reply:
interesting, got ClassNotFound error, seems cause by the removing of databaseutil in the dependency. let me sync with master and test in my local to find out if I can reproduce this... 
java.lang.NoClassDefFoundError: com/ibm/fhir/database/utils/api/IDatabaseTarget com.ibm.jbatch.container.controller.impl.ChunkStepControllerImpl

Reply:
interesting, it works well in my local. let me double check the server.xml in pipeline

Reply:
en... does reproduced in my local the same error ... trying to figure out how this can happen ...

Reply:
fixed the error by adding databaseUtil to bulkdata webapp war, and removed the export test from testng.xml of integration test because it needs COS key and service id to be configured correctly.